tests/test_command_ext.py

```diff
@@ -91,40 +91,57 @@ def test_root_command(self):
 
 @djc_test
 class TestExtensionsListCommand:
-    def test_list_command_default(self):
+    def test_list_default_extensions(self):
         out = StringIO()
         with patch("sys.stdout", new=out):
             call_command("components", "ext", "list")
         output = out.getvalue()
 
-        assert "Installed extensions:\nview\n" == output
+        assert output.strip() == "name\n====\nview"
 
-        # Check that it omits the title when verbose is 0
+    @djc_test(
+        components_settings={"extensions": [EmptyExtension, DummyExtension]},
+    )
+    def test_list_extra_extensions(self):
         out = StringIO()
         with patch("sys.stdout", new=out):
-            call_command("components", "ext", "list", "-v", "0")
+            call_command("components", "ext", "list")
         output = out.getvalue()
 
-        assert "view\n" == output
+        assert output.strip() == "name \n=====\nview \nempty\ndummy"
```

Comment:
Nit: Why is there a space after name and view here?

Reply:
The rows are formatted to have the same width as the longest row.